packages/volto/src/components/theme/View/View.jsx

```diff
@@ -244,13 +245,15 @@ class View extends Component {
               : null
           }
         />
+        <SlotRenderer name="aboveContent" content={this.props.content} />
```

Comment:
@plone/volto-team  would be interesting to hear what other slots you think should be added.
I am thinking of plone 4 viewlets managers that we had, if we need to add something to the header
and footer as well or we keep it simple for starters and add extra slots as needed.
So for sure I am hoping that others also give feedback on this work.

Reply:
I would for sure like to have built-in slots above and below the title in the title block.

Header and footer also makes sense conceptually...but then I guess we should move the existing rendering of these into slots, which is probably hard to do without it becoming a breaking change.

Reply:
We should include a list of the built-in slots in the documentation.

Reply:
@davisagli I've thought about the above and below title too. However, there's a couple of drawbacks, eg. How about the edit mode? Should they appear too there? How about if you move the title block around? Unlikely, but possible. But I agree that they will solve more problems than bring.